cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBoxHelper.java

```diff
@@ -201,7 +202,7 @@ public boolean onKey(View v, int keyCode, KeyEvent event) {
                     @Override
                     public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
                         if (actionId == EditorInfo.IME_ACTION_DONE) {
-                            Cocos2dxEditBoxHelper.closeKeyboard(index);
```

Comment:
the `setOnEditorActionListener` is a listener in UI thread, I think there is no need to called `closeKeyboardOnUiThread`